### ISSUE TITLE
ImportError: cannot import name 'Model' from 'ppm3d'

### ISSUE THREAD

**Issue by J-You** [paul-voyles/motifextraction#1]
Hi,
Thanks for your selfless open.
I try to run "python generate_clusters_for_alignment.py modelfiles/CuZr2_324.xyz" as stated in the distribution of Intermetallic Example. However, it can't import Model form ppm3d. Actually, Model is not in the source code of ppm3d. How could I reproduce this probelm? 
Thanks for your time and attention.

**Comment by jjmaldonis** [comment]
Hello! I'm happy to hear that you are using motifextraction!

I now realize that the installation instructions for this package were quite poor. I have updated the [README](https://github.com/paul-voyles/motifextraction) with much more information. Can you please try the instructions there and see if that helps? 

The `Model` class is defined the motifextraction repo [here](https://github.com/paul-voyles/motifextraction/blob/master/packages/ppm3d/tipes/cluster.py#L100) and you should be able to import it as long as your `PYTHONPATH` is set.

Please let me know if you're able to get it installed or not. If not, I am happy to help (and improve the documentation at the same time).
\- Jason

**Comment by J-You** [comment]
Thanks for your reply. Motifextraction stands out as especially effective in my work. 

I will try your improved instruction. Thanks a lot for your help.

**Comment by J-You** [comment]
Hi Jason,

I tried your detailed instruction again. But it missed the same problem "cannot import name 'Model' from 'ppm3d'".

My PYTHONPATH environment variable is: /home/user1/motifextraction/packages/ppm3d:/home/user2/motifextraction/packages/motifextraction

In fact, according to my setting of environment variable, import Model ppm3d, this ppm3d specifies ppm3d.py from distribution of /home/user1/motifextraction/packages/ppm3d/ppm3d.py.

However, class Model is from /home/user1/motifextraction/packages/ppm3d/tipes/cluster.py

Be appreciate to your help,
Jiang

**Comment by jjmaldonis** [comment]
Good catch, the `PYTHONPATH` environment variable should be set to `/home/user1/motifextraction/packages` rather than `/home/user1/motifextraction/packages/ppm3d:/home/user1/motifextraction/packages/motifextraction`. I just updated the readme to fix this.

Unfortunately I can't completely test the installation because I don't have the libraries needed to compile the ppm3d code, so I am doing a bit of guessing. I believe this will work though.

**Comment by J-You** [comment]
Thanks your improvement. 
By the way, to compile ppm3d code for other motixfextraction users, maybe you need to install gcc 4.8.5.
In my Centos7.6 systems, it works.

**Comment by J-You** [comment]
After testing, it really works. Thanks again for your help.
